sdk/src/dlob/DLOB.ts

```diff
@@ -71,6 +71,8 @@ export type MarketNodeLists = {
 	};
 };
 
+export type DLOBFilterFn = (node: DLOBNode) => boolean;
```

Comment:
Can you add a comment on this? Specifically whether the fn should return true/false to exclude the node, i think youre following js filter semantics, but better to be explicit!

Reply:
Right, I should have in the first place. I've expanded the comments and force-pushed the branch.